edx_lint/pylint/super_check.py

```diff
@@ -83,8 +83,6 @@ def visit_functiondef(self, node):
 
             try:
                 klass = next(expr.expr.infer())
-                if klass is astroid.YES:
-                    continue
```

Comment:
This doesn't seem like the right fix here.  We need to adapt to the new style of expressing uncertain inferences, not just remove the logic.

Reply:
Looking at the current documentation, i *think* it's `astroid.Uninferable`. but I don't know.

here's the [atstroid docs](https://media.readthedocs.org/pdf/astroid/latest/astroid.pdf) and I just followed what I read in chapter 1.2.

Reply:
so now that #90 has fixed this issue, I should put back in the original code here and github will handle the change, correct?

Reply:
#90 was merged about 6 hours ago; it should now work if you undo your change and rebase your branch on the current master.  I just discovered we have some docs on rebasing a pull request: https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request